lib/infra_conversion_throttler.rb

```diff
@@ -1,15 +1,24 @@
 class InfraConversionThrottler
   def self.start_conversions
     pending_conversion_jobs.each do |ems, jobs|
-      running = ems.conversion_hosts.inject(0) { |sum, ch| sum + ch.active_tasks.size }
+      running = ems.conversion_hosts.inject(0) { |sum, ch| sum + ch.active_tasks.count }
+      $log&.debug("There are currently #{running} conversion hosts running.")
       slots = (ems.miq_custom_get('Max Transformation Runners') || Settings.transformation.limits.max_concurrent_tasks_per_ems).to_i - running
+      $log&.debug("The maximum number of concurrent tasks for the EMS is: #{slots}.")
       jobs.each do |job|
-        eligible_hosts = ems.conversion_hosts.select(&:eligible?).sort_by { |ch| ch.active_tasks.size }
+        eligible_hosts = ems.conversion_hosts.select(&:eligible?).sort_by { |ch| ch.active_tasks.count }
```

Comment:
not a big fan of bringing back every conversion host to then select out the eligible ones.
to then hit the database for each of those.

would like to find a way to get `eligible` into the query and then possibly do the sum in the database too